snuba/util.py

```diff
@@ -461,7 +461,7 @@ def raw_query(request: Request, sql, client, timer, stats=None):
                                 # All queries should already be deduplicated at this point
                                 # But the query_id will let us know if they aren't
                                 query_id=query_id,
-                                with_totals=request.query.get('totals', False),
+                                with_totals=request.query.get_body().get('totals', False),
```

Comment:
Maybe expose this as a method from `ClickhouseQuery` instead?

Reply:
It is happening in #473:

https://github.com/getsentry/snuba/pull/473/files#diff-efb5d5957623ef5d07f3a3594bbab902R58

But it is on the Snuba query, right now. I think this should end up being in query settings. What do you think ?

Reply:
I don't have a strong opinion about query or settings, I think a reasonable argument could be made for either one.